sphinxcontrib/inmanta/pygments_lexer.py

```diff
@@ -71,7 +71,7 @@ def process_id(self, match):
             ("[\"]{3}([\\n]|.)*?[\"]{3}", Comment.Multiline),  # t_begin_mls
             ("\n+", Whitespace),  # t_ANY_newline
             ('!=|==|>=|<=|<|>', Token.Operator),  # t_CMP_OP
-            ('[:[\]()=,.{}]', Token.Operator),  # literals
+            ('[:[\]()=,.{}*]', Token.Operator),  # literals
```

Comment:
Is is possible to do **? Or is * also ok?

Reply:
I'm not familiar with pygments, but I assume it would be possible. I decided to just add * because : was also added as a single operator while I believe it's always used ::
Therefore I assumed it would be ok like this.

Reply:
Agreed. You can merge it in